boards/pba-d-01-kw2x/board.c

```diff
@@ -54,7 +54,7 @@ static inline void modem_clock_init(void)
     while (KW2XDRF_GPIO->PDIR & (1 << KW2XDRF_IRQ_PIN));
 }
 
-void board_init(void)
+void post_startup(void)
```

Comment:
Makes me wonder why this is not in `cpu_init()` then

Reply:
I guess back then it was the most reasonable place as cpu_init was done there and it only applies to KW2X. Adding it to cpu_init would probably be cleaner nowadays but it would need to be verified for each KW2X board/config ;)